test/e2e/framework/networking_utils.go

```diff
@@ -579,7 +580,7 @@ func (config *NetworkingTestConfig) setup(selector map[string]string) {
 	ginkgo.By("Getting node addresses")
 	ExpectNoError(WaitForAllNodesSchedulable(config.f.ClientSet, 10*time.Minute))
 	nodeList := GetReadySchedulableNodesOrDie(config.f.ClientSet)
-	config.ExternalAddrs = NodeAddresses(nodeList, v1.NodeExternalIP)
+	config.ExternalAddrs = e2enode.FirstNodeAddresses(nodeList, v1.NodeExternalIP)
```

Comment:
b/c you now have a node package the naming of the function signatures should change and should probably be done in this PR. 

e2enode.FirstAddresses()... 

This goes for all other signatures that will now have a redundant "node" in the name

Reply:
For some of the functions, changes like `WaitForNodeToBe` -> `WaitForToBe`, `CheckNodesReady`-> `CheckReady`, `NumberOfReadyNodes` -> `NumberOfReady` are ambiguous. 
How about remove part of the redundant strings but keep some that described as above?